Production/govuk_ios/Builders/ViewControllerBuilder.swift

```diff
@@ -31,4 +45,19 @@ class ViewControllerBuilder {
             viewModel: viewModel
         )
     }
+
+    func permit(permitId: String,
```

Comment:
Wonder whether the terminology should be parkingPermit, as 'permit' on its own sounds vague

Reply:
Personally, I wouldn't worry about it; it's only a placeholder for demo testing. Also, it's not necessarily only a parking permit. It could be used for fishing permits...